src/chains.jl

```diff
@@ -548,43 +574,44 @@ function header(c::Chains; section=missing)
     # Return header.
     return string(
         ismissing(c.logevidence) ? "" : "Log evidence      = $(c.logevidence)\n",
-        ismissing(start) ? "" : "Start time        = $(start)\n",
-        ismissing(stop) ? "" : "Stop time         = $(stop)\n",
-        ismissing(wall) ? "" : "Wall duration     = $(round(wall, digits=2)) seconds\n",
         "Iterations        = $(first(c)):$(last(c))\n",
         "Thinning interval = $(step(c))\n",
-        "Chains            = $(join(map(string, chains(c)), ", "))\n",
+        "Chains            = $(length(map(string, chains(c))))\n",
```

Comment:
Is it correct that with this change only the number of chains will be displayed? If this is the case, maybe change it to
```suggestion
        "Number of chains  = $(size(c, 3))\n",
```
now that only the number of chains is displayed?